### ISSUE TITLE
Is Monit vulnerable to MITM?

### ISSUE THREAD

**Issue by NOYB** [opnsense/core#2892]
Using SMTP server, either over SSL/TLS on port 465, or STARTTLS on port 587, messages go through even though there is no trusted CA for the mail server certificate.

This seems to indicate that the certificate/peer is not being validated.  Making it vulnerable to MITM.  Is this the case?  Or am I missing something?

Monit Secure Connection is enabled.  Packet capture bares this out.

**Comment by fbrendel** [comment]
Yes, the SSL certificate verification is disabled by default. See Monit [SSL Options](https://mmonit.com/monit/documentation/monit.html#SSL-OPTIONS).
I'll add these options to the General Settings.

@fichtner should we enable the verification by default?

**Comment by NOYB** [comment]
My philosophy would be for default secure so that it is required to make a conscious choice to be less secure.
Maybe a verify peer checkbox along side the secure connection enable checkbox.

Thanks

P.S.  Will Monit be using OpenSSL/LibreSSL or Certify?  Think OpenSSL/LibreSSL is more flexible for self signed certs frequently used with private mail servers.

**Comment by fichtner** [comment]
Agree with default to on as it's easily turned off after update if required. We'll add this to the release notes. Monit follows ports crypto so that's OpenSSL or LibreSSL indeed.

**Comment by fichtner** [comment]
thanks guys!
